atomicapp/utils.py

```diff
@@ -14,6 +14,18 @@
 
 logger = logging.getLogger(__name__)
 
+def printStatus(message):
+    logger.info("atomicapp.status.message="+str(message))
+
+
+def printErrorStatus(message):
+    logger.info("atomicapp.status.error.message="+str(message))
```

Comment:
I dont have any particular preference for info/debug log levels here. any acceptable log level is ok for me.